xml/x509_subjects.xml

```diff
@@ -168,6 +168,14 @@
     <param pos="0" name="os.product" value="iDRAC Linux"/>
   </fingerprint>
 
+  <fingerprint pattern="^CN=opensnort\.example.com,O=Sourcefire,emailaddress=support@sourcefire.com">
+    <description>Sourcefire Snort HTTPS console w/Service tag</description>
+    <example>CN=opensnort.example.com,O=Sourcefire,emailaddress=support@sourcefire.com</example>
+    <param pos="0" name="service.vendor" value="SOURCEFIRE"/>
```

Comment:
After looking at the NIST database and Snort website I'd recommend using `Snort` for the vendor and product. It's accurate for today (Cisco purchased Sourcefire, and Snort is no longer branded as Sourcefire) and will enable autogeneration of CPEs.  

See: https://github.com/rapid7/recog/blob/master/CONTRIBUTING.md#updating-cpes